builtin/providers/aws/resource_aws_vpc.go

```diff
@@ -344,6 +343,68 @@ func resourceAwsVpcUpdate(d *schema.ResourceData, meta interface{}) error {
 		d.SetPartial("enable_classiclink")
 	}
 
+	if d.HasChange("assign_generated_ipv6_cidr_block") && !d.IsNewResource() {
+		val := d.Get("assign_generated_ipv6_cidr_block").(bool)
```

Comment:
How do you feel about renaming this variable to something a bit more specific & self-explanatory - e.g. `toAssign`? It may help below in `if toAssign {`.